lib/src/system.dart

```diff
@@ -0,0 +1,45 @@
+import 'dart:io';
```

Comment:
I think `file_system` or `io` would be a better name for this file, "system" is too generic IMO